.github/workflows/publish-documentation.yml

```diff
@@ -41,8 +39,9 @@ jobs:
           mv docs/* .
           rm -rf docs
       - name: Compile Markdown to HTML and create artifact
+        # This is not in th repo, but in the image
```

Comment:
This is a good idea to be helpful, I think a little bit extra on what the image you referencing here can be helpful My first thought was `In what image sorry?`